### PR TITLE
Awards new

### PR DESCRIPTION
**Pull request by vilsi12**

### CODE REVIEW

**Comment by chandrikadeb7** [comment]
@vilsi12 Please don't make PR from main/master branch. It will add difficulty for your next contributions.

File: README.md [hunk 1]
```diff
@@ -133,6 +133,23 @@ Feel free to mail me for any doubts/query
 ## :handshake: Contribution
 Feel free to **file a new issue** with a respective title and description on the the [Face-Mask-Detection](https://github.com/chandrikadeb7/Face-Mask-Detection/issues) repository. If you already found a solution to your problem, **I would love to review your pull request**! 
 
+## Awards
+Awarded Runners Up position in Amdocs Innovation India ICE Project Fair
```
Comment (chandrikadeb7) [inline README.md:137]: Add link for Amdocs - https://www.amdocs.com/

File: README.md [hunk 2]
```diff
@@ -133,6 +133,23 @@ Feel free to mail me for any doubts/query
...
+
+![](Readme_images/nn.jpeg)
+## 👏Appreciation
+Selected in [Devscript Winter Of Code])(https://devscript.tech/woc/)
```
Comment (chandrikadeb7) [inline README.md:141]: Remove ')' and preview it first please if it is a valid hyperlink.

File: README.md [hunk 3]
```diff
@@ -133,6 +133,23 @@ Feel free to mail me for any doubts/query
...
+
+![](Readme_images/Devscript.jpeg)
+
+Selected in [Script Winter Of Code](https://swoc.tech/project.html)
+
+![](Readme_images/winter.jpeg)
+[Student Code-in](https://scodein.tech/)
```
Comment (chandrikadeb7) [inline README.md:148]: Student Code-In

Please insert this for Student Code-In and also write Selected in

**Comment by chandrikadeb7** [comment]
@vilsi12 Please check the Devscript hyperlink. It has an extra brace.

And please remove the last pic and link, we don't need to have the certificate picture in readme.

**Comment by vilsi12** [comment]
Yes it not completed i will do remove Apperication certificate of sci
And not put main and master branch
It will be done
Thank you
Can you add label swoc on issue please it will be great

**Comment by chandrikadeb7** [comment]
Is this PR for SWOC or DWOC?

**Comment by vilsi12** [comment]
This is pr for both but tag dwoc please add swoc i am participating in both
it will be good if you
Thank you

**Pull request merged**